CMakeLists.txt

```diff
@@ -147,10 +147,10 @@ set(IREE_EXTERNAL_TOOLING_MODULES "" CACHE STRING "")
 #
 #   iree_compiler_plugin.cmake : Included in the context of the compiler/
 #     directory before sources and bindings. Will execute with source and
-#     binary dir ${IREE_BUILD_DIR}/compiler/plugins (shared with all other 
```

Comment:
The trailing whitespaces were removed by the settings in my IDE.

Reply:
Yep, that's fine. Thanks! (I ran into the same issue)